terraform/workspace_variables/production.backend.tfvars

```diff
@@ -0,0 +1,3 @@
+storage_account_name = "s165p01tfstate"
```

Comment:
see here https://github.com/DFE-Digital/apply-for-qualified-teacher-status/pull/45/files#r881427525